patterns/base/li/li.twig

```diff
@@ -0,0 +1,4 @@
+{% for item in li %}
```

Comment:
@scottnath  - initially i had it this way but i noticed it was repeating the same content...so it didn't seem right. But i'll put it back..